### DISCUSSION TITLE
JPA와 Mybatis를 어떤 상황에서 사용해야 할지 논의가 필요할 것 같습니다.

### DISCUSSION THREAD

**Discussion by JeonTaeHyeon** [The-Dark-Nights/back-end#25] [General]
단순히 조회는 JPA를 사용하고 조회 이외의 모든 것을 Mybatis를 사용하여 처리하게 된다면, pagination이나 영속성 관리 시에 불편함이 생길 것 같은데 어떻게 생각하시나요?

**Comment by jiwon11** [comment]
가장 중요한 건 엔티티의 영속성을 유지해야 하는지가 중요할거 같습니다.
- 같은 조회라 하더라도 영속성 컨텍스트 상에서 관리하는 엔티티가 필요하다면 JPA를 사용하면 될거 같습니다.
<img width="522" alt="CleanShot 2023-08-09 at 17 04 42@2x" src="https://github.com/The-Dark-Nights/back-end/assets/19159759/dc1f570f-f01c-4ed8-b074-d2cd3ead2982">
